queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth <= '2019-01-01' AND date_of_birth >= '2016-01-01';
```

Comment:
- [ ] This date should be '2019-31-12' instead of '2019-01-01' so you include that December month in the search criteria. 

That way, the query will select the correct data 💪

Reply:
Thank you.

Reply:
![image](https://user-images.githubusercontent.com/58023246/159249184-f29863d3-fdff-4966-83d3-1eb367e830eb.png)